packer.py

```diff
@@ -48,24 +52,26 @@ def create_bins(bins: dict) -> list:
             bins[bin]['max_weight']))
     return bins_list
 
+
 def refresh_items(unfitted_items, packer):
     for item in unfitted_items:
         packer.add_item(item)
 
+
 def refresh_bin_types(bin_types, packer):
     for bin in bin_types:
         packer.add_bin(bin)
 
-def execute_packing(items_to_fit: list, bin_types: list, visualize=True, export_img=False, textualize=True) -> list:
+
+def execute_packing(items_to_fit: list, bin_types: list, visualize=False, export_img=False, textualize=False) -> list:
     fitted_items = []   # list of solutions
     tree = Tree("Packing list:", highlight=True, hide_root=True)
     while items_to_fit:
-        packing_efficacy = 0
         packer = Packer()
         refresh_bin_types(bin_types, packer)
         refresh_items(items_to_fit, packer)
         packer.pack(bigger_first=True)
-        best_bin = max(packer.bins, key=lambda b: b.efficacy)    # select the best packed bin
+        best_bin = deepcopy(max(packer.bins, key=lambda b: b.efficacy)) # select the best packed bin and copy it
         for item in best_bin.items:
             fitted_items.append((item, best_bin))           # add item+bin to solutions
```

Comment:
So `fitted_items` is an list of tuples, when each tuple is `item` from `best_bin` and `best_bin` itself. Why is that complicated ? Best bin as bin has an reference to items.

Reply:
Deepcopied, because then in the returned list we have proper number of particular bins used. 
Previously: when many items were packed into several bins of the same type - the results were showing one bin object with all of those items. 
Now: even when the items are packed in a few same type bins - the results show those bins as separate objects.

Reply:
This way it passess tests:
`test_packer_respects_bin_constraints()` and
`test_execute_packing_optimum_volume_results()`